tcs-client/pom.xml

```diff
@@ -37,7 +37,7 @@
     <dependency>
       <groupId>com.transformuk.hee.tis</groupId>
       <artifactId>tcs-api</artifactId>
-      <version>6.14.4</version>
+      <version>6.15.0</version>
```

Comment:
Just think if the dependencies are updated, then the module itself might need a version bump too.

Reply:
Sure thing :)